### ISSUE TITLE
why  train  with  your code  .Loss too high

### ISSUE THREAD

**Issue by Hjy20255** [wkentaro/fcn#88]
/scripts/learning_curve.py  ./scripts/snapshot/log.csv

loss value is too hight.  1e5
in your example log.png  .1e6
@wkentaro

**Comment by wkentaro** [comment]
Maybe it depends on the dataset you use, and the way to initialize weight.
For example, if you use larger images for training, you have larger loss value, because the loss is not averaged by image size.

**Comment by Hjy20255** [comment]
Thanks to your reply, but I am not very understand,  the segmentation  results is ok， and in your code also used pre-training model ---vgg16 model, why would loss is 1e6

**Comment by wkentaro** [comment]
Do you use example code under voc?

**Comment by wkentaro** [comment]
Could you please attach log graph? or log.csv posted to Gist.com?

**Comment by Hjy20255** [comment]
![_008](https://cloud.githubusercontent.com/assets/17898101/23856526/eeff251c-0834-11e7-9edd-feaa91f2877c.png)

**Comment by wkentaro** [comment]
It's similar to the log plot attached to README, isn't it? Where's wrong?

**Comment by Hjy20255** [comment]
Loss value without normalization  in your code ？so, loss is 1e5?  
my image is 500×500
for example loss is  4×1e5 /(500*500)=1.6 ?

**Comment by wkentaro** [comment]
Yeah.

**Comment by Hjy20255** [comment]
thank you
